server/public/model/utils.go

```diff
@@ -732,6 +732,11 @@ func IsValidHTTPURL(rawURL string) bool {
 	return true
 }
 
+func IsValidURI(rawURI string) bool {
+	_, err := url.ParseRequestURI(rawURI)
+	return err == nil
+}
+
```

Comment:
Should the code return the `error` to the user for easier debugging? I'm aware that it's uncommon in the code base, but I'm wondering if we can improve the debugging capabilities while touching the code.

Reply:
Yep good idea I'm changing this

Reply:
Ok I made the change, tried with an empty string to see the result and got:

`Error: failed to load configuration: failed to create store: unable to load on store creation: invalid config: Config.IsValid: Identity Provider Issuer URL must be a valid URI., parse "": empty url`

Not the most readable (because we do put periods in our AppErrors) but it does the trick! :) Thanks for the suggestion!